generators/ruby/codegen/src/utils/RubyUtilities.ts

```diff
@@ -99,6 +99,33 @@ export function generateGitignore(): GeneratedFile {
     return new GeneratedFile(".gitignore", RelativeFilePath.of("."), content);
 }
 
+export function generateGithubWorkflow(gemName: string, registryUrl: string, apiKeyEnvVar: string): GeneratedFile {
+    const content = `name: Publish
+
+on: [push]
+jobs:
+    publish:
+        if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+        runs-on: ubuntu-latest
+        steps:
+          - name: Checkout repo
+            uses: actions/checkout@v3
+
+          - uses: ruby/setup-ruby@v1
+            with:
+              ruby-version: 2.7
+              bundler-cache: true
+              
+          - name: Build and Push Gem
+            env:
+              GEM_HOST_API_KEY: \${{ secrets.${apiKeyEnvVar} }}
+            run: >-
+              gem build ${gemName}.gemspec
+              gem push ${gemName}-*.gem --host ${registryUrl}
```

Comment:
`${registryUrl}` same here

Reply:
Can you elaborate? (This isn't very helpful)